win32ss/gdi/ntgdi/line.c

```diff
@@ -199,15 +200,52 @@ IntGdiLineTo(DC  *dc,
 
         if (!(pbrLine->flAttrs & BR_IS_NULL))
         {
-            Ret = IntEngLineTo(&psurf->SurfObj,
-                               (CLIPOBJ *)&dc->co,
-                               &dc->eboLine.BrushObject,
-                               Points[0].x, Points[0].y,
-                               Points[1].x, Points[1].y,
-                               &Bounds,
-                               ROP2_TO_MIX(pdcattr->jROP2));
-        }
+            DPRINT1("pbrLine is not null\n");
+            DPRINT1("pbrLine->lWidth: %ld\n", pbrLine->lWidth);
+            DPRINT1("pbrLine->ulPenStyle: 0x%lX\n", pbrLine->ulPenStyle);
+            if ((pbrLine->lWidth > 1) && (pbrLine->ulPenStyle & PS_TYPE_MASK) == PS_GEOMETRIC)
+            {
+                PPATH FASTCALL PATH_CreatePath(int count);
+                BOOL FASTCALL PATH_MoveTo(PDC dc, PPATH pPath);
```

Comment:
Shouldn't they be moved to a header? Or at least at the top of the file, not in the middle of the code of a function.

Reply:
Moved to `path.h`.